extensions/spark/kyuubi-spark-connector-tpcds/src/main/scala/org/apache/kyuubi/spark/connector/tpcds/TPCDSTableUtils.scala

```diff
@@ -47,4 +55,81 @@ object TPCDSTableUtils {
     "web_sales" -> 226,
     "web_site" -> 292)
 
+  def reviseNullColumnIndex[E <: Enum[E]](table: Table, index: Int): Int = {
+    nullColumnIndexRevisers(table.getName).getReviseNullColumnIndex(index)
+  }
+
+  private val nullColumnIndexRevisers: Map[String, NullColumnIndexReviser] =
+    initNullColumnIndexRevisers()
+
+  private def initNullColumnIndexRevisers[E <: Enum[E]](): Map[String, NullColumnIndexReviser] = {
+    Table.getBaseTables.asScala
+      .filterNot(_.getName == "dbgen_version").map { table =>
```

Comment:
We excluded the table in another places as well, let extract a variable for all tpc-ds tables instead of filter out every time

Reply:
> We excluded the table in another places as well, let extract a variable for all tpc-ds tables instead of filter out every time

how about we extract `TPCDSCatalog.tables` into `TPCDSTableUtils`?

Reply:
make sense